src/frontend/components/SubscribeScheduledVideo/index.tsx

```diff
@@ -0,0 +1,174 @@
+import {
+  Box,
+  Clock,
+  Heading,
+  Notification,
+  Paragraph,
+  StatusType,
+} from 'grommet';
+import { DateTime } from 'luxon';
+import React, { useState, useRef } from 'react';
+import { Nullable } from '../../utils/types';
```

Comment:
Can you use absolute imports in place of ascending relative ones in this file please ?